package.json

```diff
@@ -47,7 +47,7 @@
     },
     "peerDependencies": {
         "@fastify/jwt": "^9.0.1",
-        "@lokalise/node-core": ">=12.0.0",
+        "@lokalise/node-core": ">=13.0.0",
```

Comment:
In theory, version 13 was supported, but as we saw issues mixing version 12 and 13, let's make it explicit

Reply:
wouldn't it be addressed by @CatchMe2 PR?

Reply:
Could be, my idea was to make it explicit but you are right, maybe it is not needed.

Will close PR and open it again if we see it is needed